prime-modules/src/main/kotlin/org/ostelco/prime/apierror/ApiErrorCodes.kt

```diff
@@ -65,6 +65,7 @@ enum class ApiErrorCode {
     FAILED_TO_STORE_PLAN,
     FAILED_TO_REMOVE_PLAN,
     ALREADY_SUBSCRIBED_TO_PLAN,
+    FAILED_TO_RENEW_SUBSCRIPTION,
     // FAILED_TO_SUBSCRIBE_TO_PLAN,
```

Comment:
If this shouldn't be enabled in the immediate  future, then remove the dead code.